### ISSUE TITLE
Multiple settings added as the same

### ISSUE THREAD

**Issue by DriesVR1232** [akumidv/tradingview-assistant-chrome-extension#88]
I have a strategy where there are multiple settings that show as the same in the optimiser but they need to be 2 seperate ones. 
![image](https://user-images.githubusercontent.com/32335909/207733250-aedb4d32-f208-4abe-ae77-599c5b568725.png)

**Comment by Buddingm** [comment]
> I have a strategy where there are multiple settings (Close %)that show as 1 setting in the optimiser but they need to be 2 seperate ones. ![image](https://user-images.githubusercontent.com/32335909/207733250-aedb4d32-f208-4abe-ae77-599c5b568725.png)

Are you sure that you use different titles for those fields?

**Comment by DriesVR1232** [comment]
Sorry for the late reply, Do you mean in the pinescript code?

**Comment by Buddingm** [comment]
Yes, you need to have different titles for all fields. Fields May not use the same title.

**Comment by DriesVR1232** [comment]
It's not my code so ill ask the dev about this

**Comment by Buddingm** [comment]
I know. I know the Original developer and use the other version a lot.

**Comment by akumidv** [comment]
Unfortunately, it is impossible to get the identifiers of the strategy parameter fields from html, respectively, fields with the same fields are indistinguishable and it is not clear how to set their values. You can try to take into account the sequence of fields, but so far I have not planned to do this logic.
